more_itertools/more.py

```diff
@@ -1004,3 +1004,53 @@ def divide(n, iterable):
         ret.append(iter(seq[start:stop]))
 
     return ret
+
+
+def always_iterable(obj):
+    """
+    Given an object, always return an iterable.
+
+    If the object is not already iterable, return a tuple containing containing
+    the object::
+
+        >>> always_iterable(1)
+        (1,)
+
+    If the object is ``None``, return an empty iterable::
```

Comment:
I found some examples (usually bare `return` clauses) where this would be helpful; I think I'm fine with the idea as you contributed it.